app/src/main/res/layout/activity_health_page_main.xml

```diff
@@ -4,9 +4,68 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".HealthPageMain">
+    android:background="@color/colorHealthHome"
+    tools:context=".MainActivity">
 
+    <Button
+        android:id="@+id/button4"
+        android:layout_width="137dp"
+        android:layout_height="156dp"
+        android:layout_marginTop="63dp"
+        android:layout_marginEnd="41dp"
+        android:text="Add +"
+        android:backgroundTint="@color/colorFinanceHome"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/button2" />
 
+    <Button
+        android:id="@+id/button3"
+        android:layout_width="137dp"
+        android:layout_height="156dp"
+        android:layout_marginStart="40dp"
+        android:layout_marginTop="63dp"
+        android:text="Workout"
+        android:backgroundTint="@color/colorFinanceHome"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/macroTrackerButton" />
 
+    <Button
+        android:id="@+id/button2"
+        android:layout_width="137dp"
+        android:layout_height="156dp"
+        android:layout_marginStart="56dp"
+        android:layout_marginTop="117dp"
+        android:layout_marginBottom="63dp"
+        android:text="Calories"
```

Comment:
Same thing as above for the `hint`s, colors, and strings. Should all be able to be resolved pretty easily through the Component Tree in Android Studio.